src/SFA.DAS.EmployerApprenticeshipsService.Domain/Models/EmployerAgreement/RemoveEmployerAgreementView.cs

```diff
@@ -9,5 +11,6 @@ public class RemoveEmployerAgreementView
         public string HashedAccountId { get; set; }
         public bool CanBeRemoved { get; set; }
         public string LegalEntityCode { get; set; }
+        public OrganisationType LegalEntitySource { get; set; }
```

Comment:
I'd call this LegalEntityOrganisationType as it will confuse other devs if they see a source being compared to a organisation type.

Reply:
I called it that to match what is on `SFA.DAS.EAS.Domain.Models.EmployerAgreement.EmployerAgreementView`. to keep them consistent. Do you want me to change that one as well? Not sure if there would be side effects.